config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # users
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+
+  # posts
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [OPTIONAL] it would be better if you use the `resources` it so easy to maintain rather than creating every single route by it self so if you are use the resources it would be like 
```Ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Hi @youmari, thank you so much for taking the time to leave this excellent review. I've implemented the change and I really appreciate your input to make my code better. Cheers!